File: app/src/main/java/g/takeru/renshu/architecture/mvp/LoginView.java
package g.takeru.renshu.architecture.mvp;

import g.takeru.renshu.kotlin.User;

/**
 * Created by takeru on 2017/10/11.
 */

public interface LoginView extends BaseView{
    void loginSuccess(User user);
    void loginFailure(String errorMsg);
}

File: app/src/main/java/g/takeru/renshu/kotlin/architecture/mvp/LoginView.kt
package g.takeru.renshu.kotlin.architecture.mvp

import g.takeru.renshu.kotlin.User

/**
 * Created by takeru on 2017/10/11.
 */

interface LoginView : BaseView {
    fun loginSuccess(user: User)
    fun loginFailure(errorMsg: String)
}
